src/js/pages/GranthIndex/components/GranthIndex.tsx

```diff
@@ -75,7 +74,7 @@ export const GranthIndex: React.FC<IGranthIndexProps> = ({ location }) => {
               <main>
                 {Object.entries(raagIndices).map(
                   ([key, { name: granthName, source, indices }]) => (
-                    <React.Fragment key={key}>
+                    <div className="granthIndex" key={key}>
```

Comment:
if this class just for the selector? if yes then prefer if data-cy like in HeaderMenu.